src/relacion-devoluciones/relacion-devoluciones.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { Query,Controller, Res, Post, Get, Put, Delete, Param, Body, HttpStatus, NotFoundException} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { RelacionDevolucionesDto } from './dto/relacion-devoluciones.dto';
+import { RelacionDevolucionesService } from './relacion-devoluciones.service';
+import { FilterDto } from '../shared/dto/filter.dto';
+
+@ApiTags('relacion-devoluciones')
+@Controller('relacion-devoluciones')
+export class RelacionDevolucionesController {
+
+    constructor(private readonly relacionDevolucionesSerivce: RelacionDevolucionesService) { }
+
+    @Post()
+    post(@Body() relacionDevolucionesDto: RelacionDevolucionesDto) {
+        return this.relacionDevolucionesSerivce.post(relacionDevolucionesDto);
+    }
+
+    @Get()
+    get(@Query() filterDto: FilterDto) {
+        return this.relacionDevolucionesSerivce.getAll(filterDto)
+    }
+
+    @Get('/:id')
+    async getById(@Res() res, @Param('id') id: string) {
+        const relacionDevoluciones = await this.relacionDevolucionesSerivce.getById(id);
+        if (!relacionDevoluciones) throw new NotFoundException("not found resource");
+        return res.status(HttpStatus.OK).json({
+            Data: relacionDevoluciones,
+            Message: "Request successfull",
+            Status: "200",
```

Comment:
Ver (1)

Reply:
Aparentemente solo quedó faltando este...
```suggestion
            Status: HttpStatus.OK,
```